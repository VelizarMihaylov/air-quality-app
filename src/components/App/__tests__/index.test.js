import React from 'react'
import { create } from 'react-test-renderer'
import { App } from '../'
import { Provider } from 'react-redux'

import store from 'state'


describe('App', () => {
  it('should render without throwing an error', () => {
    const AppRender = create(
      <Provider store={store}>
        <App
          config={{
            apiEndpoint: 'https://reallycoolapi.com/v1'
          }}
          city={null}
        />
      </Provider>
    )
    expect(AppRender).toMatchSnapshot()
  })
})
