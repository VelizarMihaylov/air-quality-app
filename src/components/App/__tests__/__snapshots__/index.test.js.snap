// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without throwing an error 1`] = `
<div
  className="App"
>
  <div>
    <h1>
      Compare Your Air
    </h1>
    <p
      className="App--p--intro"
    >
      Compare the air quality between cities in the UK.
      <br />
      Select cities to compare using the search tool below.
    </p>
    <div
      className="SearchBox"
    >
      <form
        className="SearchBox--form"
      >
        <div
          className="SearchBox--icon__spinner"
        />
        <input
          autoComplete="off"
          className="SearchBox--input"
          disabled={true}
          name="cities"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          value=""
        />
      </form>
      <ul
        className="SearchBox--datalist"
        style={
          Object {
            "display": "none",
          }
        }
      />
    </div>
  </div>
  <div />
</div>
`;
